simulation/character_file.py

```diff
@@ -130,14 +130,14 @@ def _take_advantages(self, data, builder):
     # advantages are optional
     if 'advantages' in data.keys():
       for advantage in data['advantages']:
-        builder.take_advantage('advantage')
+        builder.take_advantage(advantage)
```

Comment:
I was getting the error ``advantage is not a valid Advantage`` since every advantage showed up as being the string literal ``'advantage'``.